rust/Cargo.toml

```diff
@@ -17,3 +17,36 @@ members = [
     "utils/hex",
     "utils/run-locally",
 ]
+
+[workspace.package]
+documentation = "https://docs.hyperlane.xyz"
+edition = "2021"
+homepage = "https://hyperlane.xyz"
+license-file = "../LICENSE.md"
+publish = false
+version = "0.1.0"
+
+[workspace.dependencies]
+async-trait = { version = "0.1" }
+color-eyre = { version = "0.6" }
+config = "~0.13.3"
+derive-new = "0.5"
+ethers = { git = "https://github.com/hyperlane-xyz/ethers-rs", tag = "2023-02-10-01", features = ["abigen"] }
```

Comment:
do you think it's worth having these features at the sub-package level or workspace level? Or is the thinking for features that are used by many packages, those are specified at the workspace level